MCprep_addon/optimize_scene.py

```diff
@@ -157,17 +171,22 @@ def execute(self, context):
 			NoiseThreshold = 0.05
 		
 		if Quality:
-			MinimumSamples = Samples // 2
+			MinimumSamples = Samples // 4
 			FilterGlossy = MAX_FILTER_GLOSSY // 2
 			MaxSteps = MAX_STEPS # TODO: Add better volumetric optimizations
 
 		else:
-			MinimumSamples = Samples // 4
+			MinimumSamples = Samples // 8
 			FilterGlossy = MAX_FILTER_GLOSSY
 			MaxSteps = MAX_STEPS // 2 # TODO: Add better volumetric optimizations
 
 		# Compute device.
 		if cycles_compute_device_type == "NONE":
+			if UseScrambling:
+				UseScrambling = False # Automatic scrambling distance is useless on CPU
+				PreviewScrambling = False
+				ScramblingMultiplier = 1
+				print("Disabling Automatic Scrambling Distance as it is useless for CPU rendering!")
```

Comment:
Does it harm rendering keeping it on? Something to consider is someone who works on their file in one machine, but then renders on another (I do this often). Obviously they won't get all the best optimizations unless they ran optimize again on the other machine, but not always feasible for a render farm. If there are some settings that have no harm to CPU but positively affect GPU, I'd just always enable it.